content/accommodation.md

```diff
@@ -8,6 +8,17 @@ callout_big_1: Where to stay in Cardiff
 
 ### Recommended partner hotels
 
+The hotels listed in this section have given us preferential rates or terms or both. Their allocation of rooms for
+PyCon UK attendees is likely to be exhausted well before the conference, so **you are stongly advised to book as early
+as you can**.
+
+Note that depending upon availability you may well be able to find cheaper rates than these, but on
+less flexible terms (changes and cancellation not permitted).
+
+Other hotels are listed after these.
+
+**Important**: Cardiff is a small city, and the weekend of PyCon will be busy. The longer you leave it to book your accommodation, the more difficult it will be to get it at a good price.
```

Comment:
Surely PyCon UK is a PyCon; in the context of this event it can be just "PyCon"?